packages/playground/src/weblets/tf_nextcloud.vue

```diff
@@ -0,0 +1,252 @@
+          v-model:planetary="planetary"
+          v-model:wireguard="wireguard"
+          ref="network"
+        />
+        <input-tooltip
+          inline
+          tooltip="Click to know more about dedicated nodes."
+          href="https://manual.grid.tf/dashboard/portal/dashboard_portal_dedicated_nodes.html"
+        >
+          <v-switch color="primary" inset label="Dedicated" v-model="dedicated" hide-details />
+        </input-tooltip>
+
+        <input-tooltip inline tooltip="Renting capacity on certified nodes is charged 25% extra.">
+          <v-switch color="primary" inset label="Certified" v-model="certified" hide-details />
+        </input-tooltip>
+
+        <SelectFarmManager>
+          <SelectFarm
+            :filters="{
+              cpu: solution?.cpu,
+              memory: solution?.memory,
+              publicIp: ipv4,
+              ssd: disks.reduce((total, disk) => total + disk.size, rootFilesystemSize),
+              rentedBy: dedicated ? profileManager.profile?.twinId : undefined,
+              certified: certified,
+            }"
+            v-model="farm"
+          />
+          <SelectNode
+            v-model="selectedNode"
+            :filters="{
+              farmId: farm?.farmID,
+              cpu: solution?.cpu,
+              memory: solution?.memory,
+              ipv4: ipv4,
+              disks: disks,
+              rentedBy: dedicated ? profileManager.profile?.twinId : undefined,
+              certified: certified,
+            }"
+          />
+        </SelectFarmManager>
+      </template>
+
+      <template #env>
+        <ExpandableLayout
+          v-model="envs"
+          @add="envs.push({ key: '', value: '' })"
+          #="{ index, isRequired }"
+          :required="[0]"
+        >
+          <input-validator
+            :value="envs[index].key"
+            :rules="[
+              validators.required('Key name is required.'),
+              key => validators.isAlpha('Key must start with alphabet char.')(key[0]),
+              validators.pattern('Invalid key format.', { pattern: /^[^0-9_\s][a-zA-Z0-9_]+$/ }),
+              validators.maxLength('Key max length is 128 chars.', 128),
+            ]"
+            #="{ props }"
+          >
+            <input-tooltip tooltip="Environment key.">
+              <v-text-field label="Name" v-model="envs[index].key" :disabled="isRequired" v-bind="props" />
+            </input-tooltip>
+          </input-validator>
+
+          <input-validator
+            :value="envs[index].value"
+            :rules="[validators.required('Value is required.')]"
+            #="{ props }"
+          >
+            <input-tooltip tooltip="Environment Value.">
+              <v-textarea label="Value" v-model="envs[index].value" no-resize :spellcheck="false" />
+            </input-tooltip>
+          </input-validator>
+        </ExpandableLayout>
+      </template>
+    </d-tabs>
+
+    <template #footer-actions>
```

Comment:
```suggestion
        </SelectFarmManager>
      
    <template #footer-actions>
```

As I mentioned earlier we don't need tabs so we don't need to manage #envs tab anymore